.github/workflows/phpunit.yml

```diff
@@ -21,8 +33,6 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
```

Comment:
removed depth but no comment why. should have been separate commit anyway with explanation

Reply:
- removed depth so default value is 1. https://github.com/actions/checkout
- main reason: use git shallow clone to Improve Performance, and ci process don't need full git history.

Reply:
my message was that should express changes more better. in commit message (preferred) or pull request. because you may not be around when in the future someone questions this change.

Reply:
So i updated commit message